taskcluster/ci/client/kind.yml

```diff
@@ -30,6 +30,16 @@ tasks:
         {{ yarn --frozen-lockfile || exit 99; }} &&
         yarn test &&
         yarn package-test
+  nodejs-client-test:
```

Comment:
Nice, thanks! We'll need to update branch protection settings too to require this check once this is all merged 👍🏻